components/scream/src/physics/shoc/shoc_pblintd_height_impl.hpp

```diff
@@ -61,33 +64,32 @@ void Functions<S,D>::pblintd_height(
                       ekat::square(u(k) - s_u(nlev-1)) +
                       ekat::square(v(k) - s_v(nlev-1)) +
                       fac*(ustar*ustar)));
+
     if (in_range.any()) {
       rino(k).set(in_range,
                   ggr*(thv(k) - thv_ref)*(z(k) - s_z(nlev-1))/(s_thv(nlev-1)*vvk));
     }
 
     // Set indices_pack entry to -1 if rino(k)<ricr or
     // if global index is not in [nlev-npbl, nlev-1)
-    indices_pack.set(rino(k)<ricr || !in_range, -1);
+    indices_pack.set(rino(k)<ricr || !in_range, Kokkos::reduction_identity<Int>::max());
 
     // Local max index s.t. rino(k)>=ricr and k in [nlev-npbl, nlev-1)
     if (local_max < ekat::max(indices_pack))
       local_max = ekat::max(indices_pack);
 
   }, Kokkos::Max<Int>(max_indx));
 
+  std::cout << max_indx << std::endl;
```

Comment:
This should be removed (not desired, and won't work on gpu).

Reply:
Looks like I missed one..